projboard/tests/test_user.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from projboard.models.user import User
+
+
+NAME = "RAWAD"
+NICKNAME = "USER3"
+EMAIL = "rawad@gmail.com"
+PASSWORD = "123456"
+
+# NAME1 = "RAWAD2"
+# NICKNAME1 = "USER32"
+# EMAIL1 = "rawad2@gmail.com"
+# PASSWORD1 = "1234562"
+
+
+class TestUserModel:
+
+    @pytest.fixture
+    @pytest.mark.django_db
+    def generate_user(self):
+        user = User(email=EMAIL, password=PASSWORD, name=NAME, nickname=NICKNAME)
+        user.save()
+        return user
+
+    @pytest.mark.django_db
+    def test_new_user(self, generate_user):
```

Comment:
The test name indicates that you are testing creation of a new user.  
However, the test uses `get_user_by_nickname` and not creating a new user.

You should name your tests according to the method they test.
For example:
If you want to test `get_user`, the test name should be `test_get_user` and you can create multiple tests for the same method using informative names like:
- `get_user_new_user`
- `get_user_existing_user`
- `get_user_fail`

etc...

Reply:
Done